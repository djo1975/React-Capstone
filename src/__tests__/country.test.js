import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Country from '../components/country';

describe('Country', () => {
  it('should render correctly', () => {
    const component = renderer.create(
      <MemoryRouter>
        <Country
          name="Canada"
          flagEmoji="ðŸ‡¨ðŸ‡¦"
          continent="North America"
        />
      </MemoryRouter>,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
