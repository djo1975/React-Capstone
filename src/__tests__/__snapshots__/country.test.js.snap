// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Country should render correctly 1`] = `
<a
  className="country-link"
  href="/country/Canada"
  onClick={[Function]}
>
  <li
    className="country-item"
  >
    <h2
      className="country-name"
    >
      Canada
      <br />
      <span
        className="flag"
      >
        ðŸ‡¨ðŸ‡¦
      </span>
    </h2>
    <p
      className="country-continent"
    >
      North America
    </p>
  </li>
</a>
`;
